MediaBrowser.Controller/Persistence/IKeyframeRepository.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Jellyfin.MediaEncoding.Keyframes;
+
+namespace MediaBrowser.Controller.Persistence;
+
+/// <summary>
+/// Provides methods for accessing keyframe data.
+/// </summary>
+public interface IKeyframeRepository
+{
+    /// <summary>
+    /// Gets the keyframe data.
+    /// </summary>
+    /// <param name="itemId">The item id.</param>
+    /// <returns>IEnumerable{KeyframeData}.</returns>
```

Comment:
This should never just contain the type.
```suggestion
    /// <returns>The keyframe data.</returns>
```